src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  let discount = 0;
+
+  if (days < 3) {
+    return days * rent;
+  } else if (days < 7) {
+    discount = 20;
+
+    return days * rent - discount;
+  } else {
+    discount = 50;
```

Comment:
please make variables for 50 and 20 values because we badly understand what values it is